src/main/java/basic/TaskList.java

```diff
@@ -0,0 +1,90 @@
+package basic;
+
+import task.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    /**
+     * Constructs a new task list and initialises it with an empty ArrayList.
+     */
+    public ArrayList<Task> tasks = new ArrayList<Task>();
+
```

Comment:
I like you distinguish clearly between single-valued and multi-valued variables.